[BUG] Vite unable to resolve package - incorrect module exports

**Describe the bug**
Vite is unable to resolve the package.

**To Reproduce**
Steps to reproduce the behavior:
1. Install @pankod/refine and vite
2. Import any component anywhere
3. Run `vite`
4. Sample error:
```node_modules/vite/dist/node/chunks/dep-36bf480c.js:43807:10: error: [plugin: vite:dep-scan] Failed to resolve entry for package "@pankod/refine". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@pankod/refine". The package may have incorrect main/module/exports specified in its package.json```

**Expected behavior**
`vite` should serve the site

**Desktop (please complete the following information):**
 - MacOS Catalina
 - Safari
 - Refine v1.0.78
 - No dataProvider

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -4,7 +4,7 @@
     "description": "",
     "private": false,
     "main": "dist/index.js",
-    "module": "dist/react.esm.js",
+    "module": "dist/refine.esm.js",
     "typings": "dist/index.d.ts",
     "scripts": {
         "start": "tsdx watch --tsconfig tsconfig.json --verbose --noClean",

